### PR TITLE
LPS-153434 | master

### PR DESCRIPTION
**Pull request by JoyceWang08211**
https://issues.liferay.com/browse/LPS-153434

Note: Add pause to wait for portal deploying the config file.

### CODE REVIEW

**Comment by gaborlovasdotcom** [comment]
ci:stop

**Comment by gaborlovasdotcom** [comment]
@JoyceWang08211 Please fix the SF issue.

File: portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructureee/security/CookiesBanner.testcase [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+	@priority = "5"
+	test CookiesBannerCouldBeEnabledAndCustomizedViaOSGIFile {
+		property test.name.skip.portal.instance = "CookiesBanner#CookiesBannerCouldBeEnabledAndCustomizedViaOSGIFile";
+
+		task ("Given: a site configured to show the Banner via OSGI file") {
+			OSGiConfig.copyOSGiConfigFile(
+				osgiConfigFileBaseDir = "test/functional/com/liferay/portalweb/tests/coreinfrastructureee/security/dependencies",
+				osgiConfigFileName = "com.liferay.cookies.configuration.CookiesBannerConfiguration.config");
+
+			Pause(locator1 = "1000");
```
Comment (gaborlovasdotcom) [inline portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructureee/security/CookiesBanner.testcase:67]: Could we replace this with some dynamic pause?

**Review by gaborlovasdotcom — COMMENTED** [review]
Nice execution, and thanks for separating the existing test cases into multiple files.
We only need to replace the Pause from the test case with something more dynamic.
Currently, it breaks SF

File: portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructureee/security/CookiesBanner.testcase (hunk 1 above)
Comment (JoyceWang08211) [reply portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructureee/security/CookiesBanner.testcase:67]: I forget to add comment to the pause, I replaced the pause with while loop.

**Comment by JoyceWang08211** [comment]
ci:test:cookies:nocompile

**Comment by JoyceWang08211** [comment]
ci:test:sf

**Comment by JoyceWang08211** [comment]
> 1. CookiesConsentPanel#CookiesConsentPanelCouldBeEnabledAndCustomizedOnASitePage

CookiesConsentPanel#CookiesConsentPanelCouldBeEnabledAndCustomizedOnASitePage passes on local env with the above changes, it should be a flaky issue.

**Review by gaborlovasdotcom — APPROVED** [review]

**Comment by gaborlovasdotcom** [comment]
ci:forward

**Pull request closed without merging**